### PR TITLE
Gather all breaking changes in release notes for 2.0

### PR DESCRIPTION
**Pull request by cheenamalhotra**
P.S. This is also being worked up in parallel to be documented on Microsoft Docs.

### CODE REVIEW

**Comment by ErikEJ** [comment]
LGTM

File: release-notes/2.0/2.0.0.md [hunk 1]
```diff
@@ -39,6 +39,16 @@ New features over the 1.1 release of Microsoft.Data.SqlClient:
...
+- `SqlDataReader.GetSchemaTable()` now returns empty `DataTable` instead of returning `null` [#419](https://github.com/dotnet/SqlClient/pull/419)
+- Updated driver to perform decimal scale rounding to match SQL Server behavior [#470](https://github.com/dotnet/SqlClient/pull/470)
+- For .NET Framework applications consuming **Microsoft.Data.SqlClient**, the `SNI.dll` files previously downloaded to the `bin\x64` and `bin\x86` folders are now named `Microsoft.Data.SqlClient.SNI.x64.dll` and `Microsoft.Data.SqlClient.SNI.x86.dll` and will be downloaded to the `bin` directory, to support auto-loading in the application process [#570](https://github.com/dotnet/SqlClient/pull/570). _This change is not going to impact client applications unless a direct reference has been made to `SNI.dll` or the x86 and x64 folders._
+- Modified enclave provider interface `SqlColumnEncryptionEnclaveProvider` to be internal [#602](https://github.com/dotnet/SqlClient/pull/602) - _This change is not likely to impact customer applications since secure enclaves is a relatively new feature and they would have had to implement their own enclave provider, which is not a trivial task_.
```
Comment (karinazhou) [inline release-notes/2.0/2.0.0.md:49]: Not only SqlColumnEncryptionEnclaveProvider but also some other related classes to be internal

**Comment by johnsimons** [comment]
Should non preview releases actually roll up all the changes, not just the breaking ones? It would be much easier IMHO, thoughts?

**Comment by cheenamalhotra** [comment]
@johnsimons 

I've updated the PR accordingly.

**Pull request merged**